apps/openmw/mwmechanics/summoning.hpp

```diff
@@ -22,7 +22,7 @@ namespace MWMechanics
                             float magnitude, float remainingTime = -1, float totalTime = -1);
 
         /// To call after all effect sources have been visited
-        void process();
+        void process(bool cleanup);
```

Comment:
Could you please add some documentation for `cleanup`? I think that it will not be immediately obvious to readers unfamiliar with this issue what it does and why it is necessary.